README.rst

```diff
@@ -54,6 +54,36 @@ To play with the code, here is a starting point:
     # 16MB chunks
     multithread.ADLDownloader(adl, "", 'my_temp_dir', 5, 2**24)
 
+Progress can be tracked using a callback function in the form `track(current, total)`
+When passed, this will keep track of transferred bytes and be called on each complete chunk.
+
+Here's an example using the Azure CLI progress controller as the `progress_callback`:
+
+.. code-block:: python
+
+    from cli.core.application import APPLICATION
+
+    def _update_progress(current, total):
+        hook = APPLICATION.get_progress_controller(True)
```

Comment:
Use the keyword argument for the `True` argument (and basically any time you pass an argument where it's not self-documenting what the argument is meant for, e.g. what's `True`?).